package.json

```diff
@@ -21,8 +21,7 @@
     "start-dev": "nodemon --inspect server.js",
     "prestart-dev": "npm run lint",
     "migrations": "sequelize db:migrate",
-    "start": "node server.js",
-    "server": "babel-node server.js"
+    "start": "node --inspect server.js"
```

Comment:
Start should be simply `node server.js` (for inspect mode is start-dev command above)